izpack-core/src/test/java/com/izforge/izpack/core/data/DefaultVariablesTest.java

```diff
@@ -287,423 +291,321 @@ public void testDynamicVariablesUnset()
 
     /**
      * Tests dynamic variables with a deeper dependency
-     * @see https://jira.codehaus.org/browse/IZPACK-1182
      */
     @Test
-   public void testDependentDynamicVariables()
-   {
-        variables.add(createDynamic("depVar1", "${depVar2}"));
-        variables.add(createDynamic("depVar2", "${depVar3}"));
+    public void testDependentDynamicVariables()
+    {
         variables.set("depVar3", "depValue");
+        variables.add(createDynamic("depVar2", "${depVar3}"));
+        variables.add(createDynamic("depVar1", "${depVar2}"));
 
         assertNull(variables.get("depVar1")); // not created till variables refreshed
         variables.refresh();
-        assertEquals("check dependent variable","depValue", variables.get("depVar1"));
-   }
+        assertEquals("check dependent variable", "depValue", variables.get("depVar1"));
+    }
 
     /**
      * Tests dynamic variables with a deeper dependency
-     * @see https://jira.codehaus.org/browse/IZPACK-1182
      */
     @Test
-   public void testDependentDynamicVariables2()
-   {
-        variables.add(createDynamic("name1", "${name3}"));
+    public void testDependentDynamicVariables2()
+    {
         variables.add(createDynamic("name2", "someValue"));
         variables.add(createDynamic("name3", "${name2}"));
+        variables.add(createDynamic("name1", "${name3}"));
         assertNull(variables.get("name1")); // not created till variables refreshed
         variables.refresh();
-        assertEquals("check dependent variable","someValue", variables.get("name1"));
-   }
-
-        
-    
-   /**
-    * Tests dynamic variables with a deeper dependency and checkonce==true
-    * @see https://jira.codehaus.org/browse/IZPACK-1182
-    */
-   @Test
-   public void testDependentDynamicVariablesWithCheckOnce()
```

Comment:
The compare function of github is lying here. These test cases were note removed, but slightly changed. Github does not recognice this local changes and shows a big global change instead  :angry: